DOL.WHD.Section14c.Web/src/modules/components/accountManagementControls/accountFormTemplate.html

```diff
@@ -1,48 +1,65 @@
-
 <div class="usa-alert usa-alert-error" role="alert" ng-show="vm.savingError">
-    <div class="usa-alert-body">
-        <h3 class="usa-alert-heading">Error</h3>
-        <p class="usa-alert-text">An error occurred while saving the account. 
-            <ul ng-show="savingErrors.length > 0">
-                <li ng-repeat="error in savingErrors">{{error}}</li>
-            </u>
-        </p>
-    </div>
+  <div class="usa-alert-body">
+    <h3 class="usa-alert-heading">Error</h3>
+    <p class="usa-alert-text">An error occurred while saving the account.
+      <ul ng-show="savingErrors.length > 0">
+        <li ng-repeat="error in savingErrors">{{error}}</li>
+      </ul>
```

Comment:
One of the things I like about React/JSX is that it'll catch unclosed or mismatched tags for you.  Maybe we should add an HTML linter as part of the tests.  🤔